Allow more user customisation of appearance

dme opened the pull request:
I'm not sure that the font-lock support was working quite correctly before - it was perhaps right by accident. Anyway, here are a couple of patches to allow users to control the display of the buffers.

sulami reviewed (commented):
Thanks for the contribution, looks pretty good overall. Got some small comments.

dme commented:
On Wednesday, 2024-03-27 at 18:18:37 -07, Robin Schroer wrote:
> @sulami commented on this pull request.
>
> Thanks for the contribution, looks pretty good overall. Got some small comments.
>
>> +				   (zero-or-more blank)
>                                     (group (1+ (and (or letter
> -                                                       blank))))
> +                                                       digit
> +						       blank))))
>
> Got some indenting tabs in here that should be spaces.

Sorry. I don't seem them, so I don't notice. Switched to
`indent-tabs-mode' as nil from now on.

>> +(defcustom literate-calc-equals " => "
> +  "The string used to indicate a result."
> +  :group 'literate-calc-mode
> +  :type 'string)
> +
> +(defface literate-calc-mode-result-face '((t :inherit font-lock-comment-face))
> +  "Face used for results."
> +  :group 'literate-calc-mode)
> +
> +(defface literate-calc-mode-identifier-face '((t :inherit font-lock-variable-face))
> +  "Face used for identifiers."
> +  :group 'literate-calc-mode)
> +
>
> Could you be so kind and also add these new options to the readme, please?

Added.
-- 
Tonight I'm gonna bury that horse in the ground.

sulami commented:
Great, thank you again for your contribution!
